src/mocks/offers.js

```diff
@@ -141,13 +172,15 @@ const getOffers = (count) => {
     price: getRandomInteger(50, 700),
     householdItems: getRandomPartArray(HOUSEHOLD_ITEMS),
     owner: getRandomOwner(),
-    city: `Amsterdam`,
-    coordinates: COORDINATES[index++],
+    city,
+    coordinates,
     isFavorite: Boolean(getRandomInteger(0, 1)),
-  }));
+  });
 };
 
-const offers = getOffers(COORDINATES.length);
+const offers = Object.entries(COORDINATES).map(([city, coordinates]) =>
+  coordinates.map((coords) => getOffer(city, coords))
+).flat();
```

Comment:
Можно сразу заюзать flatMap
```suggestion
const offers = Object.entries(COORDINATES).flatMap(([city, coordinates]) =>
  coordinates.map((coords) => getOffer(city, coords))
);
```